app/models/User.js

```diff
@@ -74,6 +74,16 @@ module.exports = (sequelize, DataTypes) => {
                     if (options.fields.includes('password')) {
                         user.password = bcrypt.hashSync(user.password, 8);
                     }
+                },
+                afterCreate: record => {
```

Comment:
do not add these kind of hooks, just refetch user again after `store/register` without password